### PR TITLE
Update to 22.1.1

### PR DESCRIPTION
**Pull request by vbabenkoru**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,16 +15,16 @@ FROM prebuild as build
 RUN /bin/bash -c "mkdir -p $(go env GOPATH)/src/github.com/cockroachdb && \
     cd $(go env GOPATH)/src/github.com/cockroachdb"
 WORKDIR /go/src/github.com/cockroachdb
-RUN /bin/bash -c "git clone --branch v21.2.4 https://github.com/cockroachdb/cockroach"
+RUN /bin/bash -c "git clone --branch v22.1.1 https://github.com/cockroachdb/cockroach"
 WORKDIR /go/src/github.com/cockroachdb/cockroach
-RUN /bin/bash -c "git submodule update --init --recursive && make build && \
-    	      make install"
+RUN /bin/bash -c "git submodule update --init --recursive"
+RUN /bin/bash -c "NODE_OPTIONS=--max-old-space-size=4096 make build && make install"
```
Comment (vbabenkoru) [inline Dockerfile:21]: It was failing with an out of memory error until I set `max-old-space-size` to give node 4GB memory for compilation.

**Comment by vbabenkoru** [comment]
Compiled and verified: https://hub.docker.com/repository/docker/vbabenkoru/cockroachdb-arm64/tags?page=1&ordering=last_updated

**Comment by jrcichra** [comment]
I was successful in making ARM builds with their new bazel build system. It's working well in my ARM/AMD K8S cluster.

https://github.com/jrcichra/cockroachdb-multiarch-bazel

**Comment by vbabenkoru** [comment]
Closing since CockroachLabs started publishing arm64-based images to Docker.

**Pull request closed without merging**